mods/shields-up/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Shields Up",
+  "author": ["Friendly Neighbor", "Takarada", "Derpy Horse"],
+  "contributor": ["That Someone", "Pompano", "Tufton", "Mdsmestad"],
+  "description": "Originally formed by the Steam user Direstorm, Shields Up is the combination of five different mods: Shields Up, Friendly Technology, Alphean's Random Expansion, Pompano's Ever Expanding Galaxy, Tufton's Plugin, That Someone's Plugin, and a number of ships from Mdsmestad. Shields Up provides a plethora of new experiences with multiple new alien races, all with their own ships and technologies, as well as even more ships and outfits for the human race. Shields Up is the largest Endless Sky mod to date; play today!",
+  "thumbnail" : "https://raw.githubusercontent.com/Amazinite/Shields-Up/master/004_Shields%20Up/images/ship/battlestar.png",
+  "banner" : "https://raw.githubusercontent.com/Amazinite/Shields-Up/master/004_Shields%20Up/images/ship/battlestar.png",
+  "website": "https://github.com/Amazinite/Shields-Up",
+  "tags": [
+    "aliens",
+    "story",
+    "ships",
+    "outfits",
```

Comment:
Extra comma here. 
Note that https://amazinite.github.io/ES-Mod-Share/ isn't showing your mod because of this.
Don't blame me for the stupid json parser, blame jquery.